### PR TITLE
Install script fixes

### PR DESCRIPTION
**Pull request by cfcosta**
This PR fixes some of the errors I've got when trying to run oc+istio with the provided hack scripts.

### CODE REVIEW

**Comment by abonas** [comment]
@cfcosta I don't recall having issues with the istio scripts, so I am wondering - if this is going to be merged, will I now experience errors? :)

File: hack/istio-bookinfo-openshift.sh [hunk 1]
```diff
@@ -8,7 +10,7 @@ if [ "$?" != 0 ]; then
 fi
 
 # find istio that we installed from the install script
-INSTALL_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )/_output/istio"
+INSTALL_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )/_output/istio"
```
Comment (jmazzitelli) [inline hack/istio-bookinfo-openshift.sh]: are you sure about this?  The script is in the hack directory. Go up to the parent and there is our build's "_output" directory. Going up two parent directories seems incorrect. Am I missing something?

What we want is to reuse the _output directory that is used by the Makefile. This directory: `<root>/src/github.com/swift-sunshine/swscore/_output`

**Comment by cfcosta** [comment]
@abonas it was not working for me at all, can you check this branch to see if it does work?

File: hack/istio-bookinfo-openshift.sh (hunk 1 above)
Comment (abonas) [reply hack/istio-bookinfo-openshift.sh]: @jmazzitelli that was my ask too.

**Pull request merged**